shell/browser/native_window_views.cc

```diff
@@ -757,6 +759,20 @@ void NativeWindowViews::SetAlwaysOnTop(bool top,
     NativeWindow::NotifyWindowAlwaysOnTopChanged();
 
   widget()->SetAlwaysOnTop(top);
+
+#if defined(OS_WIN)
+  // Reset the placement flag.
+  behind_task_bar_ = false;
+  if (top) {
+    // On macOS the window is placed behind the Dock for the following levels.
+    // Re-use the same names on Windows to make it easier for the user.
+    const std::vector<std::string> levels = {
+        "floating", "torn-off-menu", "modal-panel", "main-menu", "status"};
+    behind_task_bar_ =
+        std::find(levels.begin(), levels.end(), level) != levels.end();
```

Comment:
you can use `base::Contains` from `base/stl_util.h`

Reply:
Done, thx